Components/Input/input.html

```diff
@@ -0,0 +1,166 @@
+    <link rel="stylesheet" href="/css/intro.css" />
+    <link rel="stylesheet" href="/styles/nord.min.css" />
+    <link rel="stylesheet" href="/Components/Input/input.css" />
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <nav class="nav">
+      <div class="left-nav">
+        <a href="/index.html">Endless</a>
+        <span>v1.0</span>
+      </div>
+      <div class="right-nav">
+        <a href="/Get Started/intro.html">Docs</a>
+        <a href="/Components/Avatar/avatar.html">Examples</a>
+      </div>
+    </nav>
+
+    <div class="doc-content">
+      <div class="side-bar">
+        <div class="content-heading">
+          <p>Get started</p>
+        </div>
+        <div class="sub-content">
+          <ul class="item-list">
+            <li><a href="/Get Started/intro.html">Introduction</a></li>
+            <li><a href="/Get Started/colors.html">Colors</a></li>
+            <li><a href="/Get Started/typography.html">Typography</a></li>
+            <li><a href="/Get Started/conclusion.html">Conclusion</a></li>
+          </ul>
+        </div>
+        <div class="content-heading">
+          <p><b>Components</b></p>
+        </div>
+        <div class="sub-content">
+          <ul class="item-list">
+            <li></li>
+            <li><a href="/Components/Avatar/avatar.html"> Avatar</a></li>
+            <li><a href="/Components/Badges/badges.html">Badges</a></li>
+            <li><a href="/Components/Alerts/alerts.html">Alerts</a></li>
+            <li><a href="/Components/Buttons/buttons.html">Buttons</a></li>
+            <li><a href="/Components/Cards/cards.html">Cards</a></li>
+            <li><a href="/Components/image/image.html">Images</a></li>
+            <li><a href="/Components/Input/input.html">Input</a></li>
+            <li><a href="/Components/Snackbar/snackbar.html">Snackbar</a></li>
+          </ul>
+        </div>
+      </div>
+
+      <div class="content-bar">
+        <div class="content-title">Inputs</div>
+        <div class="content-subtitle">
+          Inputs are necessary for user interaction with the website. Inputs are
+          used to take information from the user, and then save the data to the
+          server, accordingly.
+        </div>
+        <div class="content-title">Input Textbox</div>
+        <div class="content-subtitle">
+          Input textbox can be used to take information from user such as Email
+          Id, phone number, address & other descriptions
+        </div>
+        <pre>
+                <code class="plaintext">
+                    &lt;div class="input-container"&gt;
+                    &lt;p class="input-title">Username*&lt;/p&gt;
+                    &lt;input class="input-field" type="text"&gt;
+                    &lt;/div&gt;
+                </code>
```

Comment:
one tip here, if you want to add indentation to a line you can add `&emsp;` at the start of that line